forage-android/src/test/java/com/joinforage/forage/android/core/telemetry/metrics/ResponseMonitorTest.kt

```diff
@@ -161,7 +161,7 @@ class ResponseMonitorTest {
         val roundTripResponseMonitor = CustomerPerceivedResponseMonitor(vault = vaultType, userAction = userAction, mockLogger)
         roundTripResponseMonitor.start()
         roundTripResponseMonitor.end()
-        roundTripResponseMonitor.setEventOutcome(EventOutcome.SUCCESS).logResult()
+        roundTripResponseMonitor.setEventOutcome(EventOutcome.SUCCESS).setHttpStatusCode(200).logResult()
```

Comment:
I thought customer-perceived doesn't have http_status by design because there are multiple requests involved when one calls an SDK method, can we revert this?